app/services/schools/training_programme.rb

```diff
@@ -32,14 +32,10 @@ def mentors_at_school_periods
     end
 
     def mentors_at_school?
-      return school.transient_mentors_at_school if school.respond_to?(:transient_mentors_at_school)
```

Comment:
I thought they were just in the tests by accident, I missed the service itself oops